### PR TITLE
Addition of new assertion methods in TestIntegration class.

### PR DESCRIPTION
**Pull request by karandesai-96**
This PR tests certain properties from the model:
- /simulation/model/plasma/ion_number_density
- /simulation/model/plasma/level_number_density
- /simulation/model/plasma/electron_densities
- /simulation/model/plasma/tau_sobolevs
- /simulation/model/plasma/transition_probabilities
- /simulation/model/t_rads
- /simulation/model/ws
- /simulation/runner/j_estimator
- /simulation/runner/nu_bar_estimator

Assertions were not added / skipped earlier due to ongoing restructure as well as prototyping small scale implementation.

[Latest Report](http://opensupernova.org/~karandesai96/integration/doku.php?id=reports:a2fda76)

### CODE REVIEW

**Comment by karandesai-96** [comment]
I have reordered some existing tests just for the sake of uniformity - tests will be written in the same order as stored in HDF file.

**Comment by wkerzendorf** [comment]
remove all the skipif hdf things and make sure that these things work.

**Comment by karandesai-96** [comment]
@wkerzendorf I have removed all the skipif markers and made modifications in dictionary keys reflecting new HDF mechanism. Please see the hyperlink pointing to Latest Report in description.

**Comment by karandesai-96** [comment]
@wkerzendorf I have removed all the skipif markers and made modifications in dictionary keys reflecting new HDF mechanism. Please see the hyperlink pointing to Latest Report in description.

File: tardis/model.py [hunk 1]
```diff
@@ -274,7 +274,13 @@ def to_hdf(self, path_or_buf, path='', plasma_properties=None):
 
         """
         model_path = os.path.join(path, 'model')
-        properties = ['t_inner', 'ws', 't_rads', 'v_inner', 'v_outer']
+        properties = [
```
Comment (yeganer) [inline tardis/model.py]: I really don't think we want these here, right?

**Comment by yeganer** [comment]
I wouldn't change the properties saved by `to_hdf` right now as there will be major changes in the next weeks. I'd skip stuff which is currently not available in the HDF file and reenable it once the new model is done as we'd have to do it twice.

File: tardis/model.py (hunk 1 above)
Comment (karandesai-96) [inline tardis/model.py]: Well I saw a `getattr` statement blow, so I added whatever was new was tested here, for it to be stored in reference HDF file. @ftsamis is it happening automatically / easily somewhere else ?

**Comment by karandesai-96** [comment]
Then shall I again add the `skipif` markers to parameters which aren't available as of now ? @wkerzendorf thoughts ?

**Comment by wkerzendorf** [comment]
@karandesai-96 no just take everything out that you skip. I think we should not skip for now but just take the tests out and document in an issue.

**Comment by karandesai-96** [comment]
Well, most of the quantities you mentioned in the pastebin link were'nt included. Anyhow I'll keep the ones still there within and reflect this in an issue. But untested parameters are way more than you mentioned in the pastebin link.

**Pull request closed without merging**